src/style.css

```diff
@@ -1,3 +1,72 @@
 body {
   margin: 0;
+  text-transform: uppercase;
+}
+:root{
+  --blue-color:#00ACDC;
+}
+
+h1{
+  max-width: 100%;
+  font-family: 'Roboto', sans-serif;
+  line-height: 100px;
+  text-align: center;
+}
+.container{
+  display: flex;
+  background-color:white
```

Comment:
There is a missing semicolon at the end of the 'background-color' property. This could cause issues with the CSS parsing.